easybuild/easyblocks/t/tensorflow.py

```diff
@@ -152,6 +153,135 @@ def write_wrapper(self, wrapper_dir, compiler, i_mpi_root):
             adjust_permissions(wrapper, stat.S_IXUSR)
             self.log.info("Using wrapper script for '%s': %s", compiler, which(compiler))
 
+    @staticmethod
+    def get_system_libs_for_version(tf_version, as_VALID_LIBS=False):
```

Comment:
`as_VALID_LIBS` -> `as_valid_libs`

Reply:
That is intentional: In TF the name of the variable is `VALID_LIBS` and the format of the output is the same as that constant so you can diff them